backend/src/main/resources/db/migration/V3__add_likes.sql

```diff
@@ -0,0 +1,16 @@
+-- Create tables
+CREATE TABLE likes
+(
+    created_at  DATETIME(6) NOT NULL,
+    id          BIGINT      NOT NULL AUTO_INCREMENT,
+    member_id   BIGINT      NOT NULL,
+    modified_at DATETIME(6) NOT NULL,
+    template_id BIGINT      NOT NULL,
+    PRIMARY KEY (id)
+) ENGINE=InnoDB
+
+-- Add foreign key constraints
+ALTER TABLE likes
+    ADD CONSTRAINT FKa4vkf1skcfu5r6o5gfb5jf295 FOREIGN KEY (member_id) REFERENCES member (id)
+ALTER TABLE likes
+    ADD CONSTRAINT FKbrgthyu05fsswb9ysi2l79g7r FOREIGN KEY (template_id) REFERENCES template (id)
```

Comment:
⚡
저는 SQL 문법 오류 때문에 실행이 안 되네요. 

다음과 같이 바꿔 주세요~

```suggestion
) ENGINE=InnoDB;

-- Add foreign key constraints
ALTER TABLE likes
    ADD CONSTRAINT FKa4vkf1skcfu5r6o5gfb5jf295 FOREIGN KEY (member_id) REFERENCES member (id);
ALTER TABLE likes
    ADD CONSTRAINT FKbrgthyu05fsswb9ysi2l79g7r FOREIGN KEY (template_id) REFERENCES template (id);
```

Reply:
앗 뭔가 터지길래 열심히 찾고 있었는데 ㅠ 감사합니닷